development-guide.md

```diff
@@ -120,6 +144,19 @@ Do not assume order of columns in the batch files. Always read the CSV file head
     - `null_string` value is used to represent `NULL` value in all batch files.
     - `unmodified_string` value is used to indicate columns in `update_files` where the values did not change.
 
+#### Capabilities
+This operation offers several additional features, as listed below:
+
+- Datatype Mappings: Supports adjusting partner data types for each Fivetran data type.
+- Max value for columns: Provides an option to specify the maximum value for each data type.
+
+#### AlterTable
+This operation is used to communicate updates to a table. The `SchemaDiff` message within the `AlterTableRequest` parameter provides details about these updates:
+- Adding a column (`add_column`): Fivetran uses this field to provide information about a new column to be added in destination table.
+- Update Column type (`change_column_type`): This field provides information on updated type of a column in the source that needs to be reflected in a destination table.
+- Primary key updates (`new_primary_key`):  If the primary key has changed, this field lists all the new columns used in the updated primary key.
```

Comment:
```suggestion
- Primary key updates (`updated_primary_keys`):  If the primary key has changed, this field lists all the new columns used in the updated primary key.
```

Reply:
I have done the changes, please take a look.